src/renderer/mod.rs

```diff
@@ -18,8 +18,9 @@ use crate::editor::{Colors, DrawCommand, Style, WindowDrawCommand};
 use crate::settings::*;
 use cursor_renderer::CursorRenderer;
 
-#[setting_prefix = "window"]
-#[derive(Clone, SettingGroup)]
+#[derive(SettingGroup)]
+#[setting_prefix = "keyboard"]
```

Comment:
This seems definitely incorrect. Maybe a merge error?

Reply:
Yes. I think so lol.

Reply:
I think I was just brainlessly copy and pasting.

Reply:
Ok cool. Just making sure to capture, window was changed to keyboard here.